x-pack/plugins/cases/README.md

```diff
@@ -83,6 +108,21 @@ UI component:
  UI component:
   ![Configure Component][configure-img] 
 
+### `getRecentCases`
+Arguments: 
+
+|Property|Description|
+|---|---|
+|allCasesHref|`string;` href of all cases page
+|createCaseHref|`string;` 
+|getCaseDetailsHref|`(caseDetails: CaseDetailsHrefSchema) => string;` 
+|goToAllCases|`(ev: React.MouseEvent) => void;` callback for all cases link click
+|onCaseDetailsNavClick|`(caseDetails: CaseDetailsHrefSchema) => void;` callback for case details nav click
+|perPage|`number;` number of cases to show in widget
+  
+UI component:
+ ![Recent Cases Component][recent-cases-img] 
+
```

Comment:
I know is out of the scope of this PR but could you add a note that the case action type is disabled by default and the connector is not supported?

Reply:
https://github.com/elastic/kibana/pull/95176/commits/d8f0704cf19ca88573b356b18c4669be9768bc3e